apps/aehttpserver/lib/aehttpserver/web/channels/notify.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Aehttpserver.Web.Notify do
+  alias Aeutil.Serialization
+
+  def broadcast_new_transaction_in_the_pool(tx) do
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications","new_tx:" <> Base.encode16(tx.data.from_acc), %{"body" => Serialization.tx(tx, :serialize)})
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications","new_tx:" <> Base.encode16(tx.data.to_acc), %{"body" => Serialization.tx(tx, :serialize)})
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications", "new_transaction_in_the_pool", %{"body" => Serialization.tx(tx, :serialize)})
```

Comment:
I'm not aware how the `Aehttpserver.Web.Endpoint.broadcast!` function works, but we probably should use some consistent (and simpler) naming here like: "new_tx_in_pool". And also (optionally) "new_tx_for:" instead of "new_tx:".